Current date

njsfield opened the pull request:
Related #2

andrewMacmurray commented on src/elm/Main.elm:
```diff
@@ -13,7 +13,7 @@ main : Program Never Model Msg
+        , init = ( initialModel, now )
         , view = view
-        , subscriptions = \_ -> Sub.none
+        , subscriptions = always Sub.none
```
👌 I like it

andrewMacmurray commented on src/elm/Update.elm:
```diff
@@ -7,14 +7,15 @@ import Http
 import Maybe exposing (..)
```
My gut feeling with common types like `Maybe`, `Task` and `Result` and functions like `map` would be to use them like `Maybe.map` rather than just `map`, it makes the intent much clearer. Didn't spot it on the last PR but this is one of the points https://github.com/njsfield/Nearest-Asteroids-Elm/blob/7271974f50801f6675824c79b2845de6ef78a701/src/elm/Update.elm#L155

andrewMacmurray replied:
That line too you've also called `Json.Decode`, `D`, I'm wondering if that could get a little confusing with `Dict`, I'd stick to just aliasing it as `Json` (so it would be `Json.map`)

andrewMacmurray commented:
some minor comments but this is looking really good, the way you've structured it in the helpers folder is nice and tidy 👍

njsfield commented:
Thanks for the feedback :), I've made some changes- I've added some explicitly named calls (e.g Maybe.withDefault), and removed List and Maybe as imports (stumbled on [this](https://dennisreimann.de/articles/elm-modules-import.html) article that gave a reminder of the default imports)

andrewMacmurray commented:
Beautiful 😍
